planner/mapper/mapper.go

```diff
@@ -267,7 +267,7 @@ func resolveAggregates(
 	inputFields []Requestable,
 	mapping *core.DocumentMapping,
 	collectionName string,
-	schema client.SchemaDescription,
+	schema client.CollectionDefinition,
```

Comment:
todo: rename the param to `colDef` or something similar please

Reply:
Moved to https://github.com/sourcenetwork/defradb/pull/2336/files#r1504578581